ggml/src/ggml-cuda/fattn-common.cuh

```diff
@@ -640,7 +640,7 @@ static __global__ void flash_attn_combine_results(
         const float diff = meta[l].x - kqmax;
         const float KQ_max_scale = expf(diff);
         const uint32_t ftz_mask = 0xFFFFFFFF * (diff > SOFTMAX_FTZ_THRESHOLD);
-        *((uint32_t *) &KQ_max_scale) &= ftz_mask;
+        *((uint32_t *) const_cast<float *>(&KQ_max_scale)) &= ftz_mask;
```

Comment:
Why is `KQ_max_scale` declared as `const`? If we remove the `const`, we can mutate here without `const_cast`.

Reply:
I'll revert my change and remove the `const` from `KQ_max_scale` instead.